docProps.go

```diff
@@ -206,6 +207,80 @@ func (f *File) SetDocProps(docProperties *DocProperties) error {
 	return err
 }
 
+const (
+	customPropertiesFMTIDMagicNumber = "{D5CDD505-2E9C-101B-9397-08002B2CF9AE}"
```

Comment:
The const definition should be in  [templates.go](https://github.com/xuri/excelize/blob/c46099ee67bee6e5cb742e671a6d96d354a4032b/templates.go#L102-L134).